app/views/provider_interface/start_page/show.html.erb

```diff
@@ -1,5 +1,4 @@
 <%= content_for :body_class, "app-start-page" %>
-<% content_for :browser_title, t('page_titles.application') %>
```

Comment:
The provider start page does not need a unique title beyond what’s shown already (i.e. `Manage teacher training applications - GOV.UK`. Not including this line achieves the same result with less code.